web-app/css/summarystatistics.css

```diff
@@ -0,0 +1,47 @@
+/**
+ * Basic layout for tables. Use class <code>booktable</code> on the table.
```

Comment:
Just little comment about file naming. Can we maintain the camel case format? summaryStatisctics.css